code/src/utest/plugin/lsttokens/sizeadjustment/AbbTokenTest.java

```diff
@@ -75,15 +75,15 @@ public void testInvalidInputEmpty() throws PersistenceLayerException
 	public void testValidInputs() throws PersistenceLayerException
 	{
 		assertTrue(parse("Nieder�sterreich"));
-		assertEquals("Nieder�sterreich", primaryProf.get(StringKey.ABB));
+		assertEquals("Nieder�sterreich", primaryProf.get(StringKey.ABB_KR));

```

Comment:
Yikes.  Did the swap to git chew that apart?  This is mangled in every place in our test cases that used that string...